css/main.css

```diff
@@ -596,6 +624,12 @@ anim .main .main-cent .line .load.active {
   }
 }
 
+@media screen and (max-width: 1590px) {
```

Comment:
Why 1590px? This would be ultra wide category

Reply:
I didn't change this, I just added the same properties for logo1 as it was for logo. Is this okay?